src/main/java/hudson/plugins/accurev/AccuRevHiddenParametersAction.java

```diff
@@ -14,7 +14,10 @@ public AccuRevHiddenParametersAction(EnvVars values) {
     this.values = values;
   }
 
-  public void buildEnvVars(AbstractBuild<?, ?> build, EnvVars env) {
+  @Override
+  public void buildEnvironment(
+      @SuppressWarnings("deprecation") @edu.umd.cs.findbugs.annotations.NonNull Run<?, ?> run,
```

Comment:
I think it would better to find a solution rather then to pile on the suppression of deprecation.
Either choose to ignore them or find a solution in
Perhaps JetBrains annotation:
https://mvnrepository.com/artifact/org.jetbrains/annotations/16.0.3
They are under Apache 2 license so should be fine.
Or Eclipse annotation:
https://mvnrepository.com/artifact/org.eclipse.jdt/org.eclipse.jdt.annotation/2.2.100

Suppressing is definitely not the solution 😅

Reply:
Perhaps spotbugs its a better option?

Reply:
Looking at Jenkins itself, it looks like they use the javax.annotation annotations (https://github.com/jenkinsci/jenkins/search?q=NonNull&unscoped_q=NonNull).  Any reason why we wouldn't do that, too?

Reply:
I am inclined to agree with @stephenc (sorry for tagging 😇) https://blog.codefx.org/java/jsr-305-java-9/#comment-3582702088